contracts/oracles/BoundValidator.sol

```diff
@@ -29,30 +29,24 @@ contract BoundValidator is OwnableUpgradeable, BoundValidatorInterface {
     /// @notice Set this as asset address for BNB. This is the underlying for vBNB
     address public constant BNB_ADDR = 0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB;
 
-    /// @notice Emit this event when new validate configs are added
+    /// @notice Emit this event when new validation configs are added
     event ValidateConfigAdded(address indexed asset, uint256 indexed upperBound, uint256 indexed lowerBound);
 
     /// @notice Constructor for the implementation contract. Sets immutable variables.
-    /// @param vBnbAddress The address of the VBNB
+    /// @param vBnbAddress The address of the vBNB
     /// @custom:oz-upgrades-unsafe-allow constructor
     constructor(address vBnbAddress) {
-        require(vBnbAddress != address(0), "can't be zero address");
+        require(vBnbAddress != address(0), "can't be a zero address");
         vBnb = vBnbAddress;
         _disableInitializers();
     }
 
-    /**
-     * @notice Initializes the owner of the contract
-     */
-    function initialize() public initializer {
-        __Ownable_init();
-    }
-
     /**
      * @notice Add multiple validation configs at the same time
-     * @param configs config array
+     * @param configs Array of validation configs
      * @custom:access Only Governance
-     * @custom:error Zero length error thrown, if length of the array in parameter is 0
+     * @custom:error Zero length error is thrown if length of the config array is 0
+     * @custom:event Emits ValidateConfigAdded for each validation config that is successfully set
```

Comment:
According to the [guideline](https://www.notion.so/Solidity-92e9836010ff4fe5bfa3872df4f8da4e#0818446c8f78405f83cefd6ca9203bb7), aren't they `custom:events` and `custom:errors?` with the final `s`